apps/web/src/components/navigation-menu/index.tsx

```diff
@@ -226,6 +266,69 @@ function NavigationMenu(props: NavigationMenuProps) {
         borderRight: "1px solid var(--separator)"
       }}
     >
+      <Flex
+        sx={{
+          flexDirection: isTablet ? "column" : "row",
+          alignItems: "center",
+          justifyContent: isTablet ? "center" : "space-between"
+        }}
+      >
+        <Flex
+          sx={{
+            flexDirection: isTablet ? "column" : "row",
+            alignItems: "center",
+            padding: 20,
+            gap: 2
+          }}
+        >
+          <svg
+            style={{
+              width: isTablet ? 20 : isMobile ? 30 : 30,
+              height: isTablet ? 20 : isMobile ? 30 : 30
+            }}
+          >
+            <use href="#full-logo" />
+          </svg>
+          <Text
+            variant="heading"
+            sx={{
+              fontSize: 16,
+              display: isTablet ? "none" : "block"
+            }}
+          >
+            <b>Notesnook</b>
+          </Text>
+        </Flex>
+        <NavigationDropdown
+          toggleNavigationContainer={toggleNavigationContainer}
+        />
+      </Flex>
+      <Flex
+        sx={{
+          flexDirection: isTablet ? "column" : "row",
+          justifyContent: "center"
+        }}
+      >
+        {tabs.map((tab) => (
+          <NavigationItem
+            key={tab.id}
+            id={tab.id}
+            isTablet={isTablet}
+            title={tab.title}
+            icon={tab.icon}
+            selected={currentTab === tab.id}
+            onClick={() => setCurrentTab(tab.id)}
+            showTitle={false}
+          />
+        ))}
```

Comment:
I think we should create a separate component for tab items instead of reusing `NavigationItem`.